applications/nrf_desktop/doc/hid_state.rst

```diff
@@ -31,6 +31,7 @@ Configuration
 
 The |hid_state| is enabled by selecting :kconfig:`CONFIG_DESKTOP_HID_STATE_ENABLE`.
 This module is optional and turned off by default.
+To send keyboard boot reports enable option: :kconfig:`CONFIG_DESKTOP_HID_BOOT_INTERFACE_KEYBOARD`.
```

Comment:
You may mention options for both keyboard and mouse if you think it's relevant here. Please move them to a separate section in that case.